src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -2167,6 +2169,68 @@ public void testSendReminderForFeedbackSessionParticularUsers() throws Exception
         
     }
     
+    public void testSendFeedbackSessionSubmissionConfirmationEmail() throws Exception {
+        // private method. no need to check for authentication.
+        
+        FeedbackSessionAttributes fs = dataBundle.feedbackSessions.get("session1InCourse1");
+        String courseName = CoursesLogic.inst().getCourse(fs.getCourseId()).getName();
+        
+        ______TS("success: registered student");
+        
+        StudentAttributes studentToSendConfirmation = dataBundle.students.get("student5InCourse1");
+        
+        EmailWrapper emailSent = fsLogic.sendConfirmationEmailForSubmission(fs.getCourseId(), fs.getFeedbackSessionName(),
+                                                                            studentToSendConfirmation.googleId, null, null);
+
+        String errorMessage = "No feedback submission confirmation email sent to selected student "
+                              + studentToSendConfirmation.email;
+        assertNotNull(errorMessage, emailSent);
+        assertEquals(String.format(EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject(), courseName,
+                                   fs.getFeedbackSessionName()), emailSent.getSubject());
+        
+        ______TS("success: unregistered student");
+        
+        StudentAttributes unregStudent = new StudentAttributes("1", "Team0.1", "Unreg Student",
+                "                                              unreg@stud.ent", "asdf", "idOfTypicalCourse1");
```

Comment:
I think you aligned the wrong element here. The `"` remains at the left and we get a lot of whitespaces.